src/CMakeLists.txt

```diff
@@ -274,11 +272,17 @@ if(ENABLE_SANITIZER)
     target_link_options(easyeffects PRIVATE "-fsanitize=address")
 endif(ENABLE_SANITIZER)
 
-if(ENABLE_RNNOISE)
+if(LIBRNNOISE_FOUND AND ENABLE_RNNOISE)
     MESSAGE(STATUS "Enabling rnnoise")
     target_compile_definitions(easyeffects PRIVATE ENABLE_RNNOISE=1)
     target_link_libraries(easyeffects PRIVATE PkgConfig::LIBRNNOISE)
-endif(ENABLE_RNNOISE)
+endif(LIBRNNOISE_FOUND AND ENABLE_RNNOISE)
+
+if(LIBPORTAL_FOUND AND LIBPORTALQT_FOUND AND ENABLE_LIBPORTAL)
```

Comment:
Same thing here, we should not check if libportal is found here at all. Only check if the feature is enabled.